### PR TITLE
kmm: module added unit tests and runtime client

### PR DESCRIPTION
**Pull request by kononovn**

### CODE REVIEW

File: pkg/kmm/module_test.go [hunk 1]
```diff
@@ -0,0 +1,561 @@
...
+var (
+	testSchemesV1beta1 = []clients.SchemeAttacher{
+		v1beta1.AddToScheme,
+	}
+	defaultModuleName      = "testmodule"
+	defaultModuleNamespace = "testns"
+)
+
+func TestNewModuleBuilder(t *testing.T) {
+	testCases := []struct {
```
Comment (klaskosk) [inline pkg/kmm/module_test.go:24]: should also have a test case to test for a nil client

**Review by klaskosk — APPROVED** [review]
/lgtm
